### DISCUSSION TITLE
posts/2024-02-20-ai-medicine/

### DISCUSSION THREAD

**Discussion by giscus** [racheltho/rachel-blog#10] [Announcements]
# posts/2024-02-20-ai-medicine/

The enthusiasm about AI in medicine is failing to grapple with realities of the system.

https://rachel.fast.ai/posts/2024-02-20-ai-medicine/?utm_source=substack&utm_medium=email

**Comment by smohle** [comment]
I am glad I read this.
Thanks!

**Comment by jingwho** [comment]
Great analysis.

**Comment by Lauralvarez200** [comment]
All thanks to God Almighty who use Dr. Utibe to cure me from Cancer  with his herbal Medicine, For those suffering from Cancer You can contact Dr. Utibe he can also cure other sickness such as: Hiv/Aids, Stroke, Herpes Virus, Diabetes, Hepatitis B Virus, Etc. Email:: (drutibespiritualtemple@gmail.com

**Comment by maxone73** [comment]
Today, after AlphaProteo and CREME (crispr virtual lab) have been released, just to name the two most famous, and with the success they already had, would you still write the same article? As a stage 4 cancer patient and AI researcher, I am thrilled by the predictive abilities of these systems. Of course, research takes years then, but just the mere fact that you have way more clinical trials available is amazing. I am no underestimating the access problem that many patients face, I am focusing on the discovery part with my comment.

**Comment by botirkarikov-svg** [comment]
I was checking for solution in the internet then miraculously came Across Dr EBOS, the powerful herbalist that Cure Numerous individuals HPV INFECTION,then i contacted his Email: dr.eboscuringhome@gmail.com  and explained everything to him. He sent me a herbal medication which i took for 2 weeks and was cured of my HPV totally after receiving his Herbal medicine, So My friend why wait and suffer from this horrible disease when there is someone like Dr EBOS that can cure any disease such as HIV/ CANCER/ HEPATITIS B VIRUS,Ld Get Your Problem Solved. OW SPERM COUNT,ALS,LUPUS VIRUS. You Can "CONTACT Him Via Email : dr.eboscuringhome@gmail.com or via whatsapp +2348054531517
